tests/utils.rs

```diff
@@ -0,0 +1,271 @@
+            validator,
+            subject,
+            piece_id,
+            status,
+        },
+    );
+
+    if should_fail {
+        assert!(res.main_failed());
+    } else {
+        assert!(res.contains(&(
+            user,
+            IdentityEvent::ClaimValidationChanged {
+                validator,
+                subject,
+                piece_id,
+                status,
+            }
+            .encode()
+        )));
+    }
+}
+
+pub fn verify_claim_utils(
+    id_program: &Program,
+    user: u64,
+    verifier: PublicKey,
+    verifier_signature: Signature,
+    subject: PublicKey,
+    piece_id: PieceId,
+    should_fail: bool,
+) {
+    let res = id_program.send(
+        user,
+        IdentityAction::VerifyClaim {
+            verifier,
+            verifier_signature,
+            subject,
+            piece_id,
+        },
+    );
+
+    if should_fail {
+        assert!(res.main_failed());
+    } else {
+        assert!(res.contains(&(
+            user,
+            IdentityEvent::VerifiedClaim {
+                verifier,
+                subject,
+                piece_id,
+            }
+            .encode()
+        )));
+    }
+}
+
+pub fn check_claim_utils(
+    id_program: &Program,
+    user: u64,
+    subject: PublicKey,
+    piece_id: PieceId,
+    hash: [u8; 32],
+    status: bool,
+    should_fail: bool,
+) {
+    let res = id_program.send(
+        user,
+        IdentityAction::CheckClaim {
+            subject,
+            piece_id,
+            hash,
+        },
+    );
+
+    if should_fail {
+        assert!(res.main_failed());
+    } else {
+        assert!(res.contains(&(
+            user,
+            IdentityEvent::CheckedClaim {
+                subject,
+                piece_id,
+                status,
+            }
+            .encode()
+        )));
+    }
+}
+
+// META-STATE
+pub fn check_user_claims_state_utils(
+    id_program: &Program,
+    subject: PublicKey,
+    claims: BTreeMap<PieceId, Claim>,
+) {
+    match id_program.meta_state(IdentityStateQuery::UserClaims(subject)) {
+        gstd::Ok(IdentityStateReply::UserClaims(real_claims)) => {
+            if !maps_match(&real_claims, &claims) {
+                panic!("IDENTITY: User claims differ")
+            }
+        }
+        _ => {
+            unreachable!(
+                "Unreachable metastate reply for the IdentityStateQuery::UserClaims payload has occured"
+            )
+        }
+    }
+}
+
+pub fn check_claim_state_utils(
+    id_program: &Program,
+    subject: PublicKey,
+    piece_id: PieceId,
+    claim: Claim,
+) {
+    match id_program.meta_state(IdentityStateQuery::Claim(subject, piece_id)) {
+        gstd::Ok(IdentityStateReply::Claim(real_claim)) => {
+            if let Some(cl) = real_claim {
+                if claim != cl {
+                    panic!("IDENTITY: Claims differ");
+                }
+            } else {
+                panic!("IDENTITY: No such claim");
+            }
```

Comment:
We can use expect here.
```suggestion
            if claim != real_claim.expect("IDENTITY: No such claim") {
                panic!("IDENTITY: Claims differ");
            }
```